src/hooks/useToastify.tsx

```diff
@@ -0,0 +1,31 @@
+import 'react-toastify/dist/ReactToastify.css';
+
+import type { ToastOptions } from 'react-toastify';
+import { toast } from 'react-toastify';
+
+export type TostifyType = 'success' | 'error' | 'info' | 'warning';
+
+export interface ITostifyProps {
+  type?: TostifyType;
+  label: string;
+  options?: ToastOptions;
+}
+
+const defaultOptions: ToastOptions = {
+  position: 'top-right',
+  theme: 'light',
+  autoClose: 5000,
+  hideProgressBar: true,
+  closeOnClick: true,
+  pauseOnHover: true,
+  draggable: true,
+  progress: undefined,
+};
+
+
+export const UseTostify = ({ type = 'info', label, options }: ITostifyProps) =>
```

Comment:
this is a hook? `useToastfiy`

Reply:
@femacedogomes resolve this comment beforme merge.

Reply:
It's a hook. sorry for write mistake.